.drone.star

```diff
@@ -850,6 +853,7 @@ def cleanup(config):
 def versionize(version):
   if 'behat_version' in version:
     raw_version = version['behat_version']
+    return '%s' % (raw_version.replace("rc", "RC").replace("-", "")) 
```

Comment:
That doesn't make sense, it's doing exactly that 2 rows below :)

Reply:
this is without the `v`, I wanted to have a quick hack to be able to get the .json file from master

Reply:
Than you should simply return on both parts, no need to define the raw_version variable anymore.

Reply:
why are you specifying `behat_version` for every tag?
Leave it empty for all tags except the `latest` because the `latest` doesn't point to a git tag directly.

You only need it on line 15